From 06d234dcf80ee5aafc9ae4f6fae71409aac2e33c Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sat, 14 Jan 2012 23:33:24 -0500
Subject: [PATCH] tools/gvnccapture: unbreak pod syntax

For pod, the presence or absence of leading whitespace is significant.

https://bugzilla.gnome.org/show_bug.cgi?id=667943
---
 tools/gvnccapture.c |   72 +++++++++++++++++++++++++-------------------------
 1 files changed, 36 insertions(+), 36 deletions(-)

diff --git a/tools/gvnccapture.c b/tools/gvnccapture.c
index a44f012..c3fbd46 100644
--- a/tools/gvnccapture.c
+++ b/tools/gvnccapture.c
@@ -19,68 +19,68 @@
  */
 
 /*
-  =head1 NAME
+=head1 NAME
 
-  gvnccapture - VNC screenshot capture
+gvnccapture - VNC screenshot capture
 
-  =head1 SYNOPSIS
+=head1 SYNOPSIS
 
-  gvnccapture [OPTION]... [HOST][:DISPLAY] FILENAME
+gvnccapture [OPTION]... [HOST][:DISPLAY] FILENAME
 
-  =head1 DESCRIPTION
+=head1 DESCRIPTION
 
-  Capture a screenshot of the VNC desktop at HOST:DISPLAY saving to the
-  image file FILENAME. If HOST is omitted it defaults to "localhost",
-  if :DISPLAY is omitted, it defaults to ":1". FILENAME must end in a
-  known image format extension (eg ".png", ".jpeg"). Supported options
-  are
+Capture a screenshot of the VNC desktop at HOST:DISPLAY saving to the
+image file FILENAME. If HOST is omitted it defaults to "localhost",
+if :DISPLAY is omitted, it defaults to ":1". FILENAME must end in a
+known image format extension (eg ".png", ".jpeg"). Supported options
+are
 
-  =over 4
+=over 4
 
-  =item --help, -?
+=item --help, -?
 
-  Display command line help information
+Display command line help information
 
-  =item --quiet, -q
+=item --quiet, -q
 
-  Do not display information on the console when capturing the screenshot,
-  with the exception of any password prompt.
+Do not display information on the console when capturing the screenshot,
+with the exception of any password prompt.
 
-  =item --debug, -d
+=item --debug, -d
 
-  Display verbose debugging information on the console
+Display verbose debugging information on the console
 
-  =back
+=back
 
-  =head1 EXIT STATUS
+=head1 EXIT STATUS
 
-  The exit status is 0 upon successful screen capture, otherwise
-  it is a non-zero integer
+The exit status is 0 upon successful screen capture, otherwise
+it is a non-zero integer
 
-  =head1 EXAMPLES
+=head1 EXAMPLES
 
-  # gvnccapture localhost:1 desktop.png
-  Password:
-  Connected to localhost:1
-  Saved display to desktop.png
+ # gvnccapture localhost:1 desktop.png
+ Password:
+ Connected to localhost:1
+ Saved display to desktop.png
 
-  =head1 AUTHORS
+=head1 AUTHORS
 
-  Daniel P. Berrange <dan@berrange.com>
+Daniel P. Berrange <dan@berrange.com>
 
-  =head1 COPYRIGHT
+=head1 COPYRIGHT
 
-  Copyright (C) 2010 Daniel P. Berrange <dan@berrange.com>.
+Copyright (C) 2010 Daniel P. Berrange <dan@berrange.com>.
 
-  License LGPLv2+: GNU Lesser GPL version 2 or later <http://gnu.org/licenses/gpl.html>.
+License LGPLv2+: GNU Lesser GPL version 2 or later <http://gnu.org/licenses/gpl.html>.
 
-  This is free software: you are free to change and redistribute it. There is NO WARRANTY, to the extent permitted by law.
+This is free software: you are free to change and redistribute it. There is NO WARRANTY, to the extent permitted by law.
 
-  =head1 SEE ALSO
+=head1 SEE ALSO
 
-  vinagre(1)
+vinagre(1)
 
-  =cut
+=cut
 */
 
 #include <config.h>
-- 
1.7.8.3

